client/src/js/AppDescription.jsx

```diff
@@ -42,4 +41,4 @@ class AppDescription extends React.Component {
 }
 
 window.AppDescription = AppDescription;
```

Comment:
Do you still need this?

Reply:
I do, it is required for the unit testing.